src/main/java/org/candlepin/subscriptions/product/OfferingJmxBean.java

```diff
@@ -164,8 +155,8 @@ public void syncProductFromXml(String productXml) {
     }
     try {
       Object principal = ResourceUtils.getPrincipal();
-      jmsTemplate.convertAndSend(umbProperties.getProductTopic(), productXml);
       log.info("Sync of UMB product triggered over JMX by {}", principal);
+      offeringSync.syncUmbProductFromXml(productXml);
```

Comment:
Nah, since this one is part of the JMX bean, it's intended to be used for testing purposes only (and the `receive` method  may not necessarily be used in tests).